doc/howtos/ip-address-config.rst

```diff
@@ -0,0 +1,103 @@
+.. _ip-address-config:
+
+Configuring an IP Address in the Ostro |trade| OS
+######################################################
+
+The Ostro Project uses Connection Manager (ConnMan), a Linux daemon and command-line utility, for managing internet connections 
+within embedded devices. 
```

Comment:
I'd suggest we add a line to explain that this documents explains how to connect your device to a network 'manually' and _just_ mention that for proper integration in an app, people should look at the D-Bus API (and its bindings). Would that work for both of you?